report/6.Conclusion/development_practices.tex

```diff
@@ -126,3 +126,9 @@ \section{Version Control}
 
 Conclusively, the development was greatly improved by the iterative practices, both regarding the abstractive practices of development, as well as the agile principles followed through sprints, testing and continuous integration.
 The practices improves cooperation as well as velocity, especially during the final weeks of development, and should be a good foundation for a new project in the coming semesters.
+
+\section{Continous Integration}
+The group has utilised a Continous Integration tool, called TeamCity, with custom build scripts to ensure all builds and their integrity\cite{TeamCityHomepage}.
+To avoid a faulty commit being merged resulting in an erroneous build, the TeamCity server provides GitHub pull-requests with status updates via the GitHub API, to ensure that failing builds/branches were not mergeable. 
```

Comment:
```suggestion
To prevent a branch with a failed build from being merged, the TeamCity server provides GitHub pull-requests with status updates via the GitHub API, to ensure that failing branches were not mergeable. 
```

Reply:
Må ikke lave en suggestion af Lisbys suggestion - men der skal ikke være bindestreg i pull requests